src/syntax/Parser.js

```diff
@@ -355,7 +355,9 @@ export class Parser {
    * @param {Options} options
    */
   constructor(file, errorReporter = new SyntaxErrorReporter(),
-              options = traceurOptions) {
+      options = new Options()) {
+    if (!options)
```

Comment:
It happened when the caller passed `metadata.traceurOptions` but the property was `undefined` and the Scanner was not very prepared for `undefined` options.